docs/user/exams/students_guide.rst

```diff
@@ -329,6 +329,29 @@ Summary
 
    Complaining about the Assessment of a Text Exercise
 
+Grades
+^^^^^^
+- When your exam is graded, you can view your result at the exam scores page.
+- You will see your obtained points along with the maximum points you can get for individual exercises.
```

Comment:
```suggestion
- You will see your obtained points along with the maximum points you can reach for individual exercises.
```

Reply:
Probably not the best way to resolve this but I made a Google search for phrases below and got these results, so if you are not very sure that this wording is more correct then I would propose to keep it as it is. What do you think?

"maximum points you can get": About 13.800 results
"maximum points you can reach": About 8 results

Reply:
@mallgaier had a suggestion that removes this dilemma altogether so I switched to that wording. How does that look like?